docs/_sass/td-content.scss

```diff
@@ -15,6 +15,19 @@
     margin: 3.2rem 0 1.6rem;
     font-weight: 400;
   }
+  & input {
+    border: none;
+    width: 100%;
+    max-width: 300px;
+    font-family: "Open Sans";
+    border-radius: 21px;
+    background: var(--background-neumorphic);
+    padding: 8px 15px 8px 15px;
+    margin-bottom: 1.5rem;
+  }
+  & input::placeholder {
+    font-family: "font awesome 5 free", open sans, -apple-system, BlinkMacSystemFont, segoe ui, Roboto, helvetica neue, Arial, sans-serif, apple color emoji, segoe ui emoji, segoe ui symbol;
```

Comment:
@VIKAS0264 Will you update this with Qanelas Soft font?

Reply:
It would look like this 
![IMG_20230302_221840.jpg](https://user-images.githubusercontent.com/71399937/222496318-57fabc14-e3c3-4472-b4e4-fb7c0ade0930.jpg)